localization/sn.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<localizationPack name="Sénégal">
```

Comment:
Can you please switch to "Senegal" as all localization packs have an English name?